svm30/svm30.c

```diff
@@ -144,7 +144,7 @@ s16 svm_measure_signals_blocking_read(u16 *ethanol_signal, u16 *h2_signal,
         return err;
 
     sgp_get_feature_set_version(&sgp_feature_set, &sgp_product_type);
-    if (sgp_feature_set >= 0x06) {
+    if (sgp_feature_set >= 0x20) {
```

Comment:
Do we have a some constants / defines for the feature set stuff?

Reply:
We could use SGP_REQUIRE_FS from sgp_common to check against a minimal major,minor featureset

Reply:
Good point @psachs, that's exactly what they're for :dart: